streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/StickyTaskAssignor.java

```diff
@@ -72,7 +72,13 @@ private void assignStandby(final int numStandbyReplicas) {
     }
 
     private void assignActive() {
-        final int totalCapacity = sumCapacity(clients.values());
+
+        final int totalCapacity = 1;
+        
+        if(sumCapacity(clients.value()) > 0) {
+            totalCapacity = sumCapacity(clients.value();
```

Comment:
reassignment to a final? Recommend:

```
final int totalCapacity;
if (...) {
   totalCapacity = ...
} else {
  totalCapacity = ...
}
```

Reply:
Nice catch!

Reply:
@TimYagan thanks for the patch!  While I see the point here, if there's a situation the capacity across all clients is `0`when initially assigning tasks, it's probably indicative of a broader issue and maybe we should throw an `Exception` vs. setting a value of `1`.  

\cc @mjsax WDYT?